src/test/java/seedu/address/model/ModelManagerTest.java

```diff
@@ -113,33 +113,108 @@ public void setArticleDataDirectoryPath_validPath_setsEntryBookFilePath() {
         assertEquals(path, modelManager.getArticleDataDirectoryPath());
     }
 
+    @Test
+    public void setArchivesEntryBookFilePath_nullPath_throwsNullPointerException() {
+        thrown.expect(NullPointerException.class);
+        modelManager.setArchivesEntryBookFilePath(null);
+    }
+
+    @Test
+    public void setArchivesEntryBookFilePath_validPath_setsEntryBookFilePath() {
+        Path path = Paths.get("archive/book/file/path");
+        modelManager.setArchivesEntryBookFilePath(path);
+        assertEquals(path, modelManager.getArchivesEntryBookFilePath());
+    }
+
     @Test
     public void hasEntry_nullEntry_throwsNullPointerException() {
         thrown.expect(NullPointerException.class);
         modelManager.hasEntry(null);
     }
 
     @Test
-    public void hasEntry_entryNotInEntryBook_returnsFalse() {
+    public void hasListEntry_nullEntry_throwsNullPointerException() {
+        thrown.expect(NullPointerException.class);
+        modelManager.hasListEntry(null);
+    }
+
+    @Test
+    public void hasArchivesEntry_nullEntry_throwsNullPointerException() {
+        thrown.expect(NullPointerException.class);
+        modelManager.hasArchivesEntry(null);
+    }
+
+    @Test
+    public void hasFeedsEntry_nullEntry_throwsNullPointerException() {
+        thrown.expect(NullPointerException.class);
+        modelManager.hasFeedsEntry(null);
+    }
+
+    @Test
+    public void hasListEntry_entryNotInEntryBook_returnsFalse() {
```

Comment:
Missing the Archive and Feeds equivalent!

Reply:
Woops, it was a typo, this test tests all the has*Entry in one shot since it doesn't exist anywhere.